prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1531,6 +1534,16 @@ int efa_rdm_ope_post_remote_write(struct efa_rdm_ope *ope)
 		if (OFI_UNLIKELY(!pkt_entry))
 			return -FI_EAGAIN;
 
+		if (ope->fi_flags & FI_INJECT) {
+			assert(ope->iov_count == 1);
+			assert(ope->total_len <= ep->inject_size);
+			copied = ofi_copy_from_hmem_iov(pkt_entry->wiredata + sizeof(struct efa_rdm_rma_context_pkt),
+				ope->total_len, FI_HMEM_SYSTEM, 0, ope->iov, ope->iov_count, 0);
+			assert(copied == ope->total_len);
```

Comment:
This shouldn't be an assert. It should be warning and return error

Reply:
It is an assert here: https://github.com/ofiwg/libfabric/blob/main/prov/efa/src/rdm/efa_rdm_pke_utils.c#L123-L139

Reply:
https://github.com/ofiwg/libfabric/blob/387a7338492e0b875a1f1b668b818f55b1babcda/prov/efa/src/rdm/efa_rdm_pke_utils.c#L123-L139